providers/celebiogullari/provider.go

```diff
@@ -0,0 +1,31 @@
+package celebiogullari
+
+import (
+	"github.com/Rhymond/go-money"
+
+	"baklava/providers/genericparser"
+)
+
+const (
+	fistikliBaklavaURL = "https://www.celebiogullari.com.tr/celebiogullari-baklava-1-kg-paket"
+	kuruBaklavaURL     = `https://www.celebiogullari.com.tr/1-kg-paket`
```

Comment:
Denedin mi bu kodu? Calismiyor gibi geldi?

Reply:
Denedim. Şirket kuru baklava için URL'i hatalı oluşturmuş sanırım.

Denediğim sheet: https://docs.google.com/spreadsheets/d/13Ly4vUiKVP26BSorHRiO01nQhdtFv9cNy0PqoDHoCTc/edit?usp=sharing

Reply:
Denedim. Şirket kuru baklaav için böyle bir slug kullanmışlar.

Deneme sonucuna Sheet'ten bakabilirsin: https://docs.google.com/spreadsheets/d/13Ly4vUiKVP26BSorHRiO01nQhdtFv9cNy0PqoDHoCTc/edit?usp=sharing

Reply:
bunlarin hicbiri benim bilgisayarda calismiyor.

> 
![image](https://user-images.githubusercontent.com/159209/140787990-537fa925-49c9-47d0-a50a-f2fcad8ade76.png)

yurtdisindan girince farkli URL'lere izin var demekki. calistigi yer de cloud oldugu icin bu url'ler calismayacak.

mesela kuru baklava bende https://www.celebiogullari.com.tr/u/923/fistikli-kuru-baklava ve fiyat su sekilde gosteriliyor:

```
<span id="satis">23,00 <i class="fa fa-usd"></i></span>
```

bu default extractor icin sikintili olacak cunku default TRY farzediyoruz currency'i. update/edit lazim.-

> ![image](https://user-images.githubusercontent.com/159209/140788216-2b472479-c165-4e9c-90cb-a68469324054.png)

Reply:
aslinda duzeltiyorum. galiba "404" olayini sadece browser'da yapiyorlar... curl ile sayfalari cektigimde calisiyor.

Reply:
Browser'ın gönderdiği User-Agent'ı görünce önce `302 Found` veriyor. Sonra back-end bulunduğun ülkeye göre olan siteyi renderlamak için redirect ediyor. URL yapıları falan da farklı diğer ülkelerin. User-Agent olarak cURL'ü görünce doğrudan Türkiye'deymiş gibi renderlıyor. Not olarak kalmış olsun diğer baklava severlere.